frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -22,7 +62,7 @@ const ModuleUnionCard: FC<ModuleUnionCardProps> = ({
       content = <HTMLEllipsis unsafeHTML={module.captionHtml} maxLine="3" basedOn="words" />;
       break;
     case "propositions.occurrence":
-      content = <div dangerouslySetInnerHTML={{ __html: module.title }} />;
+      content = <div dangerouslySetInnerHTML={{ __html: titleCase(module.title) }} />;
```

Comment:
I think it's possible we don't need `dangerouslySetInnterHTML` for titles anymore, since they might all be raw text (no HTML tags). But if that's wrong, have we confirmed the case-changing logic does not affect HTML tags?

Reply:
Oh, I see. Do you suggest that I remove `dangerouslySetInnerHTML` and `__html` and test if it runs fine? And affecting HTML tags in which way do you mean?

Reply:
Good point @mattfredericksen. @senamn, you could test it like this:
```
content = <div>{titleCase(module.title)}</div>;
```

Reply:
To make sure all our data is safe, I would take a look at where that field is defined in our Django models, and maybe run a query like `Propositions.objects.all().values('title')`, and see if the output contains any html.

Reply:
I found one title that includes html but couldn't find an occurrence under this title:
<img width="489" alt="Screen Shot 2021-09-07 at 12 30 47 AM" src="https://user-images.githubusercontent.com/63204538/132289357-22885770-983a-451f-b68e-2a43ed7d6e1f.png">

Also, while searching for that <em>Homo sapiens</em>, I noticed this is what is shown in production:
<img width="671" alt="Screen Shot 2021-09-07 at 12 33 01 AM" src="https://user-images.githubusercontent.com/63204538/132289562-d1e51849-d0a7-4768-a9e8-8edf2eea520e.png">

Other than that, there wasn't any html included in the title.

Reply:
@senamn thanks for checking! If there's just one instance, I can just fix the data in production so that the title doesn't contain HTML.

Reply:
> @senamn thanks for checking! If there's just one instance, I can just fix the data in production so that the title doesn't contain HTML.

No worries and got it! So do you want me to wait until you fix it or just go ahead and push the updated code? And do you want me to change module.title line to be `content = <div>{titleCase(module.title)}</div>;`?

Reply:
You can go ahead and push the updated code, and yes, that should work!